skins/oasis/js/Tracking.js

```diff
@@ -671,24 +671,25 @@ jQuery(function ($) {
 			canonical = el.data('canonical');
 			if (canonical !== undefined) {
 				switch (canonical) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a "default" clause to this "switch" statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3ASwitchWithoutDefault)